developers.diem.com/src/css/custom.css

```diff
@@ -8,7 +8,11 @@ iframe {
   border: 0;
 }
 
-/* Doc Classes */
+.main-content {
```

Comment:
It looks like we should be using the grid here but we don't need to change this right now.